### ISSUE TITLE
Report Emu & AMReX hashes in the output

### ISSUE THREAD

**Issue by dwillcox** [AMReX-Astro/Emu#62]
Add a `job_info` file like Castro's where we write out the git hashes for Emu and amrex we're currently using, along with the compiler details.

**Comment by dwillcox** [comment]
Castro does this using the following script inside AMReX:

`amrex/Tools/C_scripts/makebuildinfo_C.py` -- this python script gets the git hashes and compiler information and populates functions in a `AMReX_buildInfo.cpp` file that return those to the program.

The header to include in AMReX is:

```
CEXE_headers += $(AMREX_HOME)/Tools/C_scripts/AMReX_buildInfo.H
INCLUDE_LOCATIONS +=  $(AMREX_HOME)/Tools/C_scripts
```

To see how this is done in Castro, look at how `Castro/Exec/Make.Castro` includes `Castro/Exec/Make.auto_source`.

And look at the lines following line 100 (build info section) in `Castro/Exec/Make.auto_source`.

https://github.com/AMReX-Astro/Castro/blob/main/Exec/Make.Castro

https://github.com/AMReX-Astro/Castro/blob/main/Exec/Make.auto_source

**Comment by dwillcox** [comment]
As an example, here's how Castro uses this information to write a "job_info" file into each plotfile directory:

https://github.com/AMReX-Astro/Castro/blob/fdeebc501853f578d8c3b4a852d716918bb32004/Source/driver/Castro_io.cpp#L552

**Comment by dwillcox** [comment]
Closed in #63
